cl/search/tests.py

```diff
@@ -618,7 +616,12 @@ def test_issue_727_doc_att_numbers(self):
         self.assertEqual(r.status_code, HTTP_200_OK)
 
 
-@override_settings(RELATED_USE_CACHE=False,)
+@override_settings(
```

Comment:
Can you put a comment here as to why it's important to override these for the test? Looks like you're...disabling them, maybe?

Reply:
I added a comment (default settings are optimized for run-time performance and limit the search space)